experiment/test_reporter.py

```diff
@@ -16,28 +16,87 @@
 import os
 from unittest import mock
 
-import yaml
-
+from common import yaml_utils
 from experiment import reporter
 from test_libs import utils as test_utils
 
 # pylint: disable=unused-argument,invalid-name
 
 
-def test_output_report_filestore(fs, experiment):
-    """Test that output_report writes the report and rsyncs it to the report
-    filestore."""
-    config_filepath = os.path.join(os.path.dirname(__file__), '..', 'service',
-                                   'core-fuzzers.yaml')
-    fs.add_real_file(config_filepath)
+def _setup_experiment_files(fs):
+    """Set up experiment config and core-fuzzers files and return experiment
+    config yaml."""
+    fs.add_real_file(reporter.CORE_FUZZERS_YAML)
 
-    # Get the config.
     config_filepath = os.path.join(os.path.dirname(__file__), 'test_data',
                                    'experiment-config.yaml')
     fs.add_real_file(config_filepath)
+    experiment_config = yaml_utils.read(config_filepath)
+
+    return experiment_config
+
+
+def test_output_report_filestore_with_fuzzer_variants(fs, experiment):
```

Comment:
nit: consider making these tests parametric with (fuzzer_names, expected_output_directory) pairs to get rid of redundancy.

Reply:
Done!